book/website/ethical-research/data-anonymization.md

```diff
@@ -0,0 +1,129 @@
+
+<!-- Not sure if this below section is necessary -irenekp -->
+## Anonymization Tools
+There are several open source tools that aid the process of anonymizing data. 
+* [Cornell's Anonymization Toolkit](https://sourceforge.net/projects/anony-toolkit/)
+* [sdcMicro: Statistical Disclosure Control Methods for Anonymization of Data and Risk Estimation](https://cran.r-project.org/web/packages/sdcMicro/) - an R library
+* [IQDA Qualitative Data Anonymizer](https://sourceforge.net/projects/datatool/)
+* [Amnesia](https://amnesia.openaire.eu/) by Openaire
+
+## Checklist
+> this can be done at the end or maybe as a separate checklist exercise, but please do note things down here as you go
+
+## What to learn next
+> recommended next chapters that are a good next step up
+
+## Further reading
+<!--I don't fully understand licenses (?) so not sure which of my references are ineligible -irenekp-->
+
+[The Anonymization Code by the Information Commissioner's Office, UK](https://ico.org.uk/media/1061/anonymisation-code.pdf) \
+[Privacy and data anonymization from a data scientist’s point of view](https://blog.concurlabs.com/privacy-and-data-anonymization-from-a-data-scientists-point-of-view-956226888d6b)\
+[Evaluation of Re-identification Risk for Anonymized Clinical Documents](https://www.phusewiki.org/docs/Conference%202017%20RG%20Papers/RG02.pdf) \
+[Privacy models](https://arx.deidentifier.org/overview/privacy-criteria/)\
+[Protecting Privacy when Disclosing Information: k-Anonymity and Its Enforcement through Generalization and Suppression](https://epic.org/privacy/reidentification/Samarati_Sweeney_paper.pdf) 
+
+
+## Definitions/glossary
+> Link to the glossary here or copy in key concepts/definitions that readers should be aware of to get the most out of this chapter
+
+## Bibliography
+
+[1] https://www.whitehouse.gov/sites/whitehouse.gov/files/omb/memoranda/2010/m10-23.pdf [Appendix]\
+[2] https://dataprivacylab.org/projects/identifiability/paper1.pdf \
+[3] https://iapp.org/media/pdf/resource_center/Guide_to_Anonymisation.pdf \
+[4] https://bmcmedinformdecismak.biomedcentral.com/articles/10.1186/1472-6947-12-66 \
+[5] https://www.oreilly.com/radar/podcast/how-privacy-preserving-techniques-can-lead-to-more-robust-machine-learning-models/ \
+[6] https://arxiv.org/abs/cs/0610105 \
+[7] http://www.ehealthinformation.ca/wp-content/uploads/2014/08/2009-De-identification-PA-whitepaper1.pdf \
+[8] https://bmcmedinformdecismak.biomedcentral.com/articles/10.1186/1472-6947-12-66
+  
```

Comment:
general style guide comment - it's helpful to keep sentences on different lines in the code: this doesn't affect the code on the page, but makes it easier to review